src/lib/components/DeckGLMap/layers/wells/wellsLayer.js

```diff
@@ -0,0 +1,31 @@
+import { CompositeLayer } from "@deck.gl/core";
+import { GeoJsonLayer } from "@deck.gl/layers";
+
+export default class WellsLayer extends CompositeLayer {
+    renderLayers() {
+        var properties = {
+            id: "wells-shadow",
```

Comment:
You can use `getSubLayerProps` (See [the doc](https://deck.gl/docs/developer-guide/custom-layers/composite-layers#forwarding-properties)) to propagate common attributes such as `opacity`, `pickable` etc and concatenate the `id` with the parent `id`.